.vscode/scripts/runtime/k3d/build_vehicleapp.sh

```diff
@@ -20,11 +20,11 @@ BUILD_ARGS=
 if [ -n "$HTTP_PROXY" ]; then
     echo "Building image with proxy configuration"
 
```

Comment:
Does this check make sense?

Reply:
Yes, without this, you do not know are you working behind proxy or not and do you need to pass proxy settings

Reply:
Yes, but imho this is not necessary at all: If you pass an env var (of the outside env) via --build-arg <env var name> to docker build and it is not defined, nothing happens - the variable is not set in the container. It's like the --build-arg would not be present at all. If the variable is defined (outside), it will be defined the same way inside as outside. (Same for option -e  with docker run.)
So, if you don't need the output "Building with or w/o proxy", the if would be superfluous.

Reply:
yep @BjoernAtBosch  is right here. We are already using this principle when building the devcontainer. The Port and the Host are always passed even if not set. So please remove this If-condition